src/commands/args.ts

```diff
@@ -42,6 +42,11 @@ export const cdxIndexCommandArgs = (yarg: yargs.Argv) => {
       describe:
         "Use plain urlkey, do not convert to SURT form (Sort-friendly URI Reordering Transform)",
       type: "boolean",
+    })
+    .option("fields", {
+      alias: "f",
+      describe: "fields to include in index",
```

Comment:
```suggestion
      describe: "comma-separated list of fields to include in index",
```

Since we're not using yarg's `array` type, might be good to be explicit about the expected format of the input

Reply:
Hm, maybe should use the array type, will look.